### ISSUE TITLE
Editor field not saving data in new version 3.5.8.1

### ISSUE THREAD

**Issue by messycodegirl** [reduxframework/redux-framework#2655]
Seems this issue has returned in Version 3.5.8.1

https://github.com/reduxframework/redux-framework/issues/2072

Any suggestions to a fix would be much appreciated. 

Thanks so much!

**Comment by kprovance** [comment]
I am unable to reproduce on my own with this limited information.  Please provide us with a support URL, and specifically the text you are entering which is not saving.

**Comment by messycodegirl** [comment]
Hi, 

Thanks for the response. I am currently working locally so sorry I cannot provide a url at this time. The text I am entering is nothing special. Just me banging my head on the keyboard lol =P

Is there anything else I can give you to help out? 

Thanks again!

**Comment by kprovance** [comment]
No, that's the information I need.  If you cannot provide it, there isn't much else I can do.  No one here has been able to reproduce,  It might be something in your config or setup.

**Comment by freetruman** [comment]
Hi, I'm having the same problem, specifically with the Sorter field, when using the 'data' argument. If there's no 'data' value, and I enter some sample text items using 'options', I can change the values, reload the page, and they'll have saved successfully. But when I set 'data' to posts, pages, categories, or whatever, saving doesn't work -- when I drag posts over to the enabled column, hit Save, and then reload the options page, the enabled list is empty.

Here are my 'fields' options.

        'fields'     => array(
	   array(
                'id'		=> 'my_post_selector',
                'type'		=> 'sorter',
                'title'		=> 'Manually Selected List of Posts',
                'data'		=> ['disabled' => 'posts'],
	   ),
    	),

I'm using  v3.5.8.1. And to repeat, in the database '_options' table, I see the entry for my Redux options, and the posts I dragged over to enabled *are* successfully stored there, but when I go to edit them in the Redux panel, the enabled list is blank. I've fiddled around with various configuration options, to no avail.
